mne/evoked.py

```diff
@@ -778,6 +779,48 @@ def plot_joint(self, times="peaks", title='', picks=None,
                            exclude=exclude, show=show, ts_args=ts_args,
                            topomap_args=topomap_args)
 
+    def animate(self, ch_type='mag', times=None, frame_rate=None,
+                butterfly=False, blit=True):
+        """Make animation of evoked data as topomap timeseries. Animation can
+        be paused/resumed with left mouse button. Left and right arrow keys can
+        be used to move backward or forward in time
+
+        Parameters
+        ----------
+        ch_type : str | None
+            Channel type to plot. Accepted data types: 'mag', 'grad', 'eeg'.
+            If None, first available channel type from ['mag', 'grad', 'eeg']
+            is used. Defaults to None.
+        times : array of floats | None
+            The time points to plot. If None, 10 evenly spaced samples are
+            calculated over the evoked time series. Defaults to None.
+        frame_rate : int | None
+            Frame rate for the animation in Hz. If None,
+            frame rate = sfreq / 10. Defaults to None.
+        butterfly : bool
+            Whether to plot the data as butterfly plot under the topomap.
+            Defaults to False.
+        blit : bool
+            Whether to use blit to optimize drawing. In general, it is
+            recommended to use blit in combination with ``show=True``. If you
+            intend to save the animation it is better to disable blit.
+            Defaults to True.
+
+        Returns
+        -------
+        fig : instance of matplotlib figure
```

Comment:
I don't think that's accurate. It's just the figure that is created for the axes objects.